packs/hpe-icsp/actions/icsp_buildplan_apply.yaml

```diff
@@ -0,0 +1,28 @@
+---
+  name: "icsp_buildplan_apply"
+  runner_type: "run-python"
+  description: "Apply Buidlplan to Specified Servers"
+  enabled: true
+  entry_point: "icsp_buildplan_apply.py"
+  parameters:
+    buildplan_uris:
+      type: "array"
+      description: "Comma seperated list of build plan URIs"
+      required: true
+      position: 1
```

Comment:
so far i've been using it to control the order or the parameter within the st2 ui.